deploy/base/model.yaml

```diff
@@ -0,0 +1 @@
+file: embeddings_us_filtered_2023_1_26_22_15_41.csv
```

Comment:
Who updates this file to the latest version?

Reply:
This is like the bigtable version, so oncall would send a PR to update it as a release step.

Reply:
ok